frontend/src/hooks/api/actions/useInviteTokenApi/useInviteTokenApi.ts

```diff
@@ -0,0 +1,73 @@
+import { useCallback } from 'react';
+import { useSWRConfig } from 'swr';
+import { url as revalidateUrl } from 'hooks/api/getters/useInviteTokens/useInviteTokens';
+import useAPI from '../useApi/useApi';
+import type {
+    ICreateInvitedUser,
+    IPublicSignupTokenCreate,
+    IPublicSignupTokenUpdate,
+} from 'interfaces/publicSignupTokens';
+
+const URI = 'api/admin/invite-link/tokens';
+
+export const useInviteTokenApi = () => {
+    const { makeRequest, createRequest, errors, loading } = useAPI({
+        propagateErrors: true,
+    });
+    const { mutate } = useSWRConfig();
```

Comment:
I think this should not be the concern of the useInviteTokenApi, but it should be the concern of the component actually using the API. You might not always need to refresh the data when doing the API call. Perhaps you do in this instance, but I think having the component decide if it wants data to refresh is a cleaner separation